### PR TITLE
DOCS-569 - Add Cloud SIEM course content

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds the self-paced courses for Cloud SIEM fundamentals and admin to documentation as the following articles:
* Introduction to Cloud SIEM for Analysts
* Introduction to Cloud SIEM for Administrators

It also moves some Cloud SIEM articles to different locations.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-569](https://sumologic.atlassian.net/browse/DOCS-569)

### CODE REVIEW

File: docs/cse/get-started-with-cloud-siem/intro-for-administrators.md [hunk 1]
```diff
@@ -0,0 +1,511 @@
+---
+id: introduction-to-cloud-siem-for-administrators
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/cse/get-started-with-cloud-siem/intro-for-administrators.md]: Absolutely, @kimsauce! I fixed it with commit "[Fix file IDs](https://github.com/SumoLogic/sumologic-documentation/pull/4851/commits/078a2d13478649a1f4faaac0bf1c2865b5fbdc0a)".

File: docs/cse/get-started-with-cloud-siem/intro-for-analysts.md [hunk 2]
```diff
@@ -0,0 +1,443 @@
+---
+id: introduction-to-cloud-siem-for-analysts
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/cse/get-started-with-cloud-siem/intro-for-analysts.md]: Fixed with commit "[Fix file IDs](https://github.com/SumoLogic/sumologic-documentation/pull/4851/commits/078a2d13478649a1f4faaac0bf1c2865b5fbdc0a)".

File: docs/cse/get-started-with-cloud-siem/intro-for-analysts.md [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+* Who is behind the event?
+* What assets did the event affect?
+* Where did the event occur?
+* When did the event occur?
+* Why did the event occur?
+* How did the event occur? 
+
+When signals cluster together, Cloud SIEM uses their tactics and techniques to name the insights they generate. The insight's name can point you to how the event occurred, or why the adversary is behaving that way. For example, a tactic name like discovery or persistence shows the reasons the adversary has. Similarly, tactic names like initial access or execution can tell you a little about the methods the adversary used. These names are just starting points, however, and you may need to revise your hypotheses as you continue your investigations.
+
+Example: An insight is named "Discovery with Execution"." Why did the event occur? Probably so the adversary could discover your information. How did the event occur? By using an executable file or a similar technique.
```
Comment (kimsauce) [inline docs/cse/get-started-with-cloud-siem/intro-for-analysts.md:189]:
```suggestion
Example: An insight is named "Discovery with Execution". Why did the event occur? Probably so the adversary could discover your information. How did the event occur? By using an executable file or a similar technique. 
```

File: docs/cse/get-started-with-cloud-siem/intro-for-administrators.md [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+    1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). Click **Records** at the top of the screen. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the main Sumo Logic menu, select **Cloud SIEM > Records**. You can also click the **Go To...** menu at the top of the screen and select **Records**. 
+    1. In the **Filters** bar, select **Metadata Source Category** from the dropdown.
+    1. Select the **is** operator from the dropdown.
+    1. Type the source category you used in [Enable data forwarding for an HTTP source](#enable-data-forwarding-for-an-http-source).
+    1. You should see a failed record. This record failed because the custom JSON log message does not have a log or ingest mapping associated with it. Although the log message was forwarded from Sumo Logic to Cloud SIEM, it did not successfully complete the parse, map, and enrich steps of the pipeline. We'll fix this in the next sections.
+
+#### Tips and tricks
+
+* If you get an error after running the CURL command in step 2, make sure your quotes are straight. Copy and pasting the command sometimes changes the formatting of these quote marks into curly quotes.
```
Comment (kimsauce) [inline docs/cse/get-started-with-cloud-siem/intro-for-administrators.md:147]:
```suggestion
* If you get an error after running the CURL command in step 2, make sure your quotation marks are straight. Copy and pasting the command sometimes changes the formatting of these quote marks into curly quotes.
```

File: docs/cse/get-started-with-cloud-siem/intro-for-administrators.md [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+    1. In the **Filters** bar, select **Metadata Source Category** from the dropdown.
+    1. Select the **is** operator from the dropdown.
+    1. Type the source category you used in [Enable data forwarding for an HTTP source](#enable-data-forwarding-for-an-http-source).
+    1. You should see a record with the IP address and threat name parsed properly.
+
+Although the log message and method of ingestion was identical, the log message failed to parse into a Cloud SIEM record in [Send a log message to Cloud SIEM](#send-a-log-message-to-cloud-siem) because the log and ingest mappings weren't configured. After we configured the log and ingest mappings, the new log messages forwarded to Cloud SIEM successfully completed the parse, map, and enrich steps to become a record.
+
+#### Tips and tricks
+
+* If you get an error after running the CURL command, make sure your quotes are straight. Copy and pasting the command sometimes changes the formatting of these quote marks into curly quotes.
```
Comment (kimsauce) [inline docs/cse/get-started-with-cloud-siem/intro-for-administrators.md:248]:
```suggestion
* If you get an error after running the CURL command, make sure your quotation marks are straight. Copy and pasting the command sometimes changes the formatting of these quote marks into curly quotes.
```

**Review by kimsauce — APPROVED** [review]
Looks perfect! Just a couple minor things.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Comment by jpipkin1** [comment]
Closing this PR because merge conflicts are too complex. Replaced by PR #4864.

**Pull request closed without merging**